first_analysis.py

```diff
@@ -27,17 +27,30 @@
 plt.ylabel('Survived')
 plt.show()
 
-# running a logistic regression (survived ~ gender + fare)
-mod = smf.logit('survived_dummy ~ gender + fare', data = df).fit()
-
-display(mod.summary())
-
 
 # running a logistic regression (survived ~ gender + fare + age)
 mod = smf.logit('survived_dummy ~ gender + fare + age', data = df).fit()
 
 display(mod.summary())
 
 
+# creating a 3D plot (survived ~ gender + fare)
+survived = df[df['survived_dummy'] == 1]
+died = df[df['survived_dummy'] == 0]
+
+fig2 = plt.figure(figsize = (12, 8))
+ax2 = fig2.add_subplot(projection='3d')
+ax2.scatter(survived['gender_dummy'], survived['fare'],survived['survived_dummy'],
+           color = 'red', label = 'survived')
```

Comment:
Maybe just for neatness:

```suggestion
ax2.scatter(survived['gender_dummy'],
                    survived['fare'],
                    survived['survived_dummy'],
                    color='red', 
                    label='survived')
```